README.md

```diff
@@ -63,6 +63,12 @@ jobs:
     # Allowed values: "true" or "false".
     # Default: false
     lean4checker: false 
+
+    
+    # Enable GitHub caching.
+    # Allowed values: "true" or "false".
+    # If use-github-cache input is not provided, the action will use GitHub caching by default.
+    use-github-cache: true
```

Comment:
We're inconsistent in this code block whether we specify the default value of not. I think the intention is that the displayed values are the defaults?

Reply:
Since this is tangentially related to the `use-github-cache` input, I will move this documentation improvement to #45 and merge this PR.